### ISSUE TITLE
Use `application-name` Parameter for `eb init` 

### ISSUE THREAD

**Issue by chasen-bettinger** [CircleCI-Public/aws-elastic-beanstalk-orb#4]
## Describe Request:

Currently, the code in this orb runs eb init using the environment-name parameter. 

`eb init <<parameters.environment-name>> -r $AWS_DEFAULT_REGION -p <<parameters.platform-version>>`

This is problematic because the aws cli eb init runs on the application level. In Elastic Beanstalk, environments and applications are not the same thing. We utilize the Elastic Beanstalk structure to organize the same code together under one application (e.g. one application has three different environments (test, stage, prod).

## Recommendation:

The job takes in a new parameter: `application-name`. So, now users can have more granular control over where they are deploying their applications.

`eb init <<parameters.application-name>> -r $AWS_DEFAULT_REGION -p <<parameters.platform-version>>`

## Supporting Documentation Links:

https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb3-init.html

**Comment by minarc** [comment]
Good request, application-name and environment-name are different.

`eb init application-name`

**Comment by KyleTryon** [comment]
We'll get this resolved asap. Thank you 👍

**Comment by minarc** [comment]
I think `eb use <environment>` could be replace environment-name.

**Comment by tylerdmace** [comment]
@KyleTryon Any update on this? Really wanna use this orb but can't because of this small problem.

EDIT: Opened #12 to address this. Please let me know what I can do to help this along :)

**Comment by KyleTryon** [comment]
Pr is now awaiting review. @tylerdmace I took your changes into a new branch, in hindsight, I should have kept them separate.

We had to make some changes to migrate the tests to a new AWS account and update the sample project. The change looks good but I think we will need to release this as a major version change.

If anyone would like to view/comment/review: https://github.com/CircleCI-Public/aws-elastic-beanstalk-orb/pull/14
